Build failure: choose-gui

### Steps To Reproduce

Steps to reproduce the behavior:
1. build `choose-gui`

### Build log

```
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/zw5j1zjfgv50bw9n9520p2ky2sx32xls-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
no configure script, doing nothing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
warning: unhandled Target key DefaultVariant
warning: unhandled Target key SupportedTargets
warning: unhandled Target key VersionMap
warning: unhandled Target key Variants
warning: unhandled Target key DebuggerOptions
warning: unhandled Product key iOSSupportVersion
warning: unhandled Target key DefaultVariant
warning: unhandled Target key SupportedTargets
warning: unhandled Target key VersionMap
warning: unhandled Target key Variants
warning: unhandled Target key DebuggerOptions
warning: unhandled Product key iOSSupportVersion
warning: unhandled Target key DefaultVariant
warning: unhandled Target key SupportedTargets
warning: unhandled Target key VersionMap
warning: unhandled Target key Variants
warning: unhandled Target key DebuggerOptions
warning: unhandled Product key iOSSupportVersion
Build settings from the command line:
    ARCHS=arm64
    SYMROOT=./output

=== BUILD TARGET choose OF PROJECT choose WITH CONFIGURATION Release ===

Check dependencies

Write auxiliary files
/bin/mkdir -p /var/empty/Library/Developer/Xcode/DerivedData/choose-ewaktwiqwuyfrcfcxsdyvajhyuum/Build/Intermediates/choose.build/Release/choose.build

** BUILD FAILED **

The following build commands failed:
(0 failures)
```

### Notify maintainers

@heywoodlh 

### Metadata

```sh
 - system: `"aarch64-darwin"`
 - host os: `Darwin 24.1.0, macOS 15.1`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.18.5`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
 ```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/ch/choose-gui/package.nix
+++ b/pkgs/by-name/ch/choose-gui/package.nix
@@ -3,7 +3,7 @@
   stdenv,
   fetchFromGitHub,
   xcbuild,
-  darwin,
+  apple-sdk_11,
 }:
 
 stdenv.mkDerivation rec {
@@ -19,11 +19,11 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ xcbuild ];
 
-  buildInputs = [ darwin.apple_sdk.frameworks.Cocoa ];
+  buildInputs = [ apple-sdk_11 ];
 
   buildPhase = ''
     runHook preBuild
-    xcodebuild -arch ${stdenv.hostPlatform.darwinArch} -configuration Release SYMROOT="./output" build
+    xcodebuild -configuration Release SYMROOT="./output" HOME="$(mktemp -d)" build
     cp ./output/Release/choose choose
     runHook postBuild
   '';

